### PR TITLE
Feature adding new user

### PR DESCRIPTION
**Pull request by Ravennod**

### CODE REVIEW

File: MedicalDocManagment.WebUI/app/user-add/user-add.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿<div class="form-group">
+    <label>UserName</label>
+    <input class="form-control" name="username" [(ngModel)]="user.userName" />
```
Comment (artempanko) [inline MedicalDocManagment.WebUI/app/user-add/user-add.component.html]: camel case

File: MedicalDocManagment.UsersDAL/Models/UserModel.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    public class UserModel
+    {
+        public string UserName { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string FirstName { get; set; }
+        public string SecondName { get; set; }
+        public string LastName { get; set; }
+        public string Position { get; set; }
+
```
Comment (artempanko) [inline MedicalDocManagment.UsersDAL/Models/UserModel.cs]: empty line

File: MedicalDocManagment.WebUI/Controllers/api/AdminController.cs [hunk 3]
```diff
@@ -32,6 +33,22 @@ public IHttpActionResult GetUsers()
...
+                FirstName = userModel.FirstName,
+                SecondName = userModel.SecondName,
+                LastName = userModel.LastName,
+                Position = userModel.Position,
+                IsActive = true
+            };
+            var result = await UsersManager.CreateAsync(user, userModel.Password);
+
+            return result;
+        }
```
Comment (artempanko) [inline MedicalDocManagment.WebUI/Controllers/api/AdminController.cs]: empty line

File: MedicalDocManagment.WebUI/app/user-add/http.service.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+    constructor(private http: Http) { }
+
+    postData(obj: User) {
+        const body = JSON.stringify(obj);
+
+        let headers = new Headers({ 'Content-Type': 'application/json;charset=utf-8' });
+
+        return this.http.post('/api/Admin/AddUser', body, { headers: headers })
+            .map((resp: Response) => resp.json())
```
Comment (artempanko) [inline MedicalDocManagment.WebUI/app/user-add/http.service.ts]: put tabs

**Review by artempanko — APPROVED** [review]
approve with minor comments

**Comment by KostetskyiYaroslav** [comment]
In `_label_` tags don't contain a `_for_` attribute.
It isn't very important, but anyway.
Approve

**Review by KostetskyiYaroslav — APPROVED** [review]

**Pull request merged**